source/modules/MEI.xml

```diff
@@ -3067,22 +3070,29 @@
         <valItem ident="circle">
           <desc>Enclosing circle.</desc>
         </valItem>
-        <valItem ident="dblwhole">
-          <desc>Enclosing "fences".</desc>
-        </valItem>
       </valList>
     </content>
   </macroSpec>
   <macroSpec ident="data.NOTEHEADMODIFIER.pat" module="MEI" type="dt">
     <desc>Captures text rendered in the center of the notehead.</desc>
     <content>
       <rng:choice>
-        <rng:data type="string">
-          <rng:param name="pattern">centertext\((A|B|C|D|E|F|G)(f|♭|n|♮|s|♯)?\)</rng:param>
-        </rng:data>
-        <rng:data type="string">
-          <rng:param name="pattern">centertext\(H(s|♯)?\)</rng:param>
-        </rng:data>
+        <rng:choice>
+          <rng:data type="string">
+            <rng:param name="pattern">centertext\((A|B|C|D|E|F|G)(f|♭|n|♮|s|♯)?\)</rng:param>
+          </rng:data>
+          <rng:data type="string">
+            <rng:param name="pattern">centertext\(H(s|♯)?\)</rng:param>
+          </rng:data>
+        </rng:choice>
+        <rng:choice>
+          <rng:data type="string">
+            <rng:param name="pattern">centertext\((Do|Re|Mi|Fa|So|La|Ti|Si)\)</rng:param>
```

Comment:
Should this also allow “Ut”?

Reply:
As far as I know it is not used in modern solmization (and haven't really been used since the 17th century). As written above I simply followed the corresponding SMuFL ranges.

Reply:
I think "Ut" should be allowed for use in pre-modern notation.